src/wp-includes/class-wp-theme-json.php

```diff
@@ -3236,17 +3236,19 @@ public function merge( $incoming ) {
 		 * some values provide exceptions, namely style values that are
 		 * objects and represent unique definitions for the style.
 		 */
-		$style_nodes = static::get_styles_block_nodes();
-		foreach ( $style_nodes as $style_node ) {
-			$path = $style_node['path'];
```

Comment:
It looks like this is the only piece of data used from the expensive `get_styles_block_nodes()` calls, so it makes sense not to call that function just to get this path. 👍

Reply:
Make sense